README.md

```diff
@@ -97,15 +112,190 @@ A quick look at the top-level files and directories you'll see in a Gatsby proje
 
 13. **`README.md`**: A text file containing useful reference information about your project.
 
-## 🎓 Learning Gatsby
+14. **`__generated__`**: Where TypeScript typings are generated for your GraphQL queries. You can use these files for strongly typing your App
+
+15. **`babel.config.js`**: [Babel configurations](https://babeljs.io/docs/en/configuration#babelrcjson) for you app. This is where you can change the targeted browsers.
+
+16. **`cypress`**: End to End(e2e) tests using Cypress. Most of the cenarios are covered in here. Add your custom flows to avoid regressions
```

Comment:
```suggestion
16. **`cypress`**: End to End(e2e) tests using Cypress. Most of the scenarios are covered here. Add your custom flows to avoid regressions
```